CityPuzzle/Classes/GameLogic.cs

```diff
@@ -235,6 +236,11 @@ where InRange(puzzle)
             }
 
         }
+        public static Position GetTargetPosition()
```

Comment:
this method could be non static solving the issue of quest latitude and longitude being non static as well

Reply:
I can access GameLogic fields from QuestPage with a static method. Otherwise, QuestPage must have GameLogic object.

Reply:
QuestPage should have a global GameLogic object because it is used over the whole page. I will fix it in the future.